zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/SystemContext.java

```diff
@@ -110,12 +113,31 @@ private void validateConfiguration() {
 
     validateExperimentalConfigs(cluster, brokerCfg.getExperimental());
 
+    validateExporters(brokerCfg.getExporters());
+
     final var security = brokerCfg.getNetwork().getSecurity();
     if (security.isEnabled()) {
       validateNetworkSecurityConfig(security);
     }
   }
 
+  private void validateExporters(final Map<String, ExporterCfg> exporters) {
+    final Set<Entry<String, ExporterCfg>> entries = exporters.entrySet();
+    final Stream<Entry<String, ExporterCfg>> badExporters =
+        entries.stream()
+            .filter(
+                entry ->
+                    entry.getValue().getClassName() == null
+                        || entry.getValue().getJarPath() == null);
```

Comment:
:question: I missed this in the previous review. Is `jarPath` mandatory configuration? It can be null, right?

Reply:
By reading the exporters section in `broker.yaml.template` and `broker.standalone.yaml.template` I would say so by looking at the elasticsearch and opensearch examples that only state the `className`. I'll remove the references to `jarPath`